stdlib/ballerina-builtin/src/main/ballerina/log/Package.md

```diff
@@ -4,35 +4,36 @@ This package provides a basic API for logging.
 
 ### Loggers 
 
-Each package in Ballerina has its own, dedicated logger. A log record contains the timestamp, log level, package name and the log message. The `printError()` function takes an optional `error` record apart from the log message. A sample log record logged from the `foo` package would look as follows:
-
-`2018-04-09 11:33:21,300 ERROR [foo] - This is an error log.`
+Each package in Ballerina has its own dedicated logger. A log record contains the timestamp, log level, package name, and the log message. The `printError()` function takes an optional `error` record apart from the log message. A sample log record logged from the `foo` package would look as follows:
+```bash
+2018-04-09 11:33:21,300 ERROR [foo] - This is an error log.
+```
 
 ### Log Output
 
 Logs are written to the `stderr` stream (i.e., the console) by default in order to make the logs more container friendly.
 
 To publish the logs to a file, redirect the `stderr` stream to a file.
-```
+```bash
 $ ballerina run program.bal 2> b7a-user.log
 ```
 
 ### Log Levels
 
-This package provides functions to log at the `WARN`, `ERROR`, `INFO`, `DEBUG` and `TRACE` levels. By default, all log messages are logged to the console at the `INFO` level. In addition to these, there are two other levels named `OFF` and `ALL`. The `OFF` log level turns off logging and the `ALL` log level allows all log levels. Log level can be configured through the Config API.
+This package provides functions to log at the `WARN`, `ERROR`, `INFO`, `DEBUG`, and `TRACE` levels. By default, all log messages are logged to the console at the `INFO` level. In addition to these, there are two other levels named `OFF` and `ALL`. The `OFF` log level turns off logging, and the `ALL` log level allows all log levels. The log level can be configured through the Config API.
 
 The `b7a.log.level` configuration key can be used to configure the log level for the log API (i.e., for all the packages). The following can be provided in a configuration file.
-```
+```toml
 b7a.log.level="<LOG_LEVEL>"
 ```
 
 Log level can also be configured through a CLI parameter as follows:
-```
+```bash
 $ ballerina run program.bal -e b7a.log.level=<LOG_LEVEL>
 ```
 
 Log levels can be configured for packages either through a configuration file or through a CLI parameter as follows:
-```
+```bash
```

Comment:
This is not bash specific. This is for both bash and config file. So just formatting this as a code block would do.